src/core/contractInterpreters/ENSToken_0xc183.json

```diff
@@ -0,0 +1,162 @@
+                    "filters": {
+                        "eventName": "Approval",
+                        "owner": "{userAddress}"
+                    },
+                    "defaultValue": "unknown"
+                }
+            }
+        },
+        "claimTokens": {
+            "action": "claimed",
+            "exampleDescriptionTemplate": "{userName} {action} {amountClaimed} {tokenName}",
+            "exampleDescription": "Carlos.eth claimed 0.1 ENS",
+            "keywords": {
+                "amountClaimed": {
+                    "key": "amount",
+                    "filters": {
+                        "eventName": "Claim",
+                        "claimant": "{userAddress}"
+                    },
+                    "defaultValue": "unknown"
+                },
+                "tokenName": {
+                    "key": "contractSymbol",
+                    "filters": {
+                        "eventName": "Transfer",
+                        "to": "{userAddress}"
+                    },
+                    "defaultValue": "unknown"
+                }
+            }
+        },
+        "delegate": {
+            "action": "delegated",
+            "exampleDescriptionTemplate": "{userName} {action} to {toDelegate}",
+            "exampleDescription": "Carlos.eth delegated to Alice.eth",
+            "keywords": {
+                "toDelegate": {
+                    "key": "toDelegate",
+                    "filters": {
+                        "eventName": "DelegateChanged"
+                    },
+                    "defaultValue": "unknown"
+                }
+            }
+        },
+        "delegateBySig": {
+            "action": "delegated",
+            "exampleDescriptionTemplate": "{userName} {action} to {toDelegate}",
+            "exampleDescription": "Carlos.eth delegated to Alice.eth",
+            "keywords": {
+                "toDelegate": {
+                    "key": "toDelegate",
+                    "filters": {
+                        "eventName": "DelegateChanged"
+                    },
+                    "defaultValue": "unknown"
+                }
+            }
+        },
+        "permit": {
+            "action": "approved",
+            "exampleDescriptionTemplate": "{userName} {action} {spender} as spender",
+            "exampleDescription": "Carlos.eth approved Alice.eth as spender",
+            "keywords": {
+                "spender": {
+                    "key": "spender",
+                    "filters": {
+                        "eventName": "Approval"
+                    },
+                    "defaultValue": "unknown spender"
+                },
+                "tokenName": {
+                    "key": "contractSymbol",
+                    "filters": {
+                        "eventName": "Approval"
+                    },
+                    "defaultValue": "unknown"
+                }
+            }
+        },
+        "transfer": {
```

Comment:
When tokens get transferred, the delegate gets changed too. We need an interpretation for that action also,

There's a section in the contribution docs regarding "multi-sided transaction"